src/app/malaria-101/activity-3/activity-3.component.ts

```diff
@@ -120,8 +123,16 @@ export class OddOneOutComponent implements OnInit {
         this._questionNumber++;
         if (this._questionNumber === 5) {
             this.activityComplete = true;
-            this._sharedData.customAlert(this.language.alerts.title, this.solutions, 'info');
+            swal({
+              title: this.language.alerts.title,
+              html: this.solutions,
+              type: 'info',
+            }).then( res => { if (!this.completed) {
+            this._badgeService.updateBadgeNumber(2).subscribe(response => response)
+          }});
+            // this._sharedData.customAlert(this.language.alerts.title, this.solutions, 'info');
```

Comment:
remove the commented out code if this will not be used.

Reply:
Removed it.